### ISSUE TITLE
Not working via Terraform Registry

### ISSUE THREAD

**Issue by karabasosman** [Azure/terraform-azurerm-postgresql#19]
It's getting release tar.gz file not found when you're getting this module from Terraform Registry directly.

``
module "postgresql" {
  source  = "Azure/postgresql/azurerm"
  version = "1.4.0"
}
``

**Error : NOT FOUND https://github.com/Azure/terraform-azurerm-postgresql/archive/v1.4.0.tar.gz**

**Comment by ramonparisg** [comment]
I am having the same problem here!

**Comment by Doctorbal** [comment]
Likewise

```
Could not download module "artifactory_db" (main.tf:154) source code from
"https://github.com/Azure/terraform-azurerm-postgresql/archive/v1.4.0.tar.gz//*?archive=tar.gz":
bad response code: 404.
```

**Comment by tanwarsatya** [comment]
is this down or removed, any maintainer ??

**Comment by yupwei68** [comment]
Hi @karabasosman , thanks for opening this issue. Sorry for the delayed response. Currently, this problem has been fixed, you can download version "1.4.0" from Terraform Registry. But the code has been out of date, so it could not pass "terraform validate". I'll upgrade and release a up-to-time version soon.
